recipe/meta.yaml

```
@@ -37,16 +36,16 @@ requirements:
     - fftw 3.3.9
```

https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L270
```suggestion
    - fftw {{ fftw }}
```